### PR TITLE
[release-4.13] OCPBUGS-28253: Use DSR load balancing in kube-proxy

### PR DESCRIPTION
**Pull request by sebsoto**
Manual cherry-pick of #2041

### CODE REVIEW

**Comment by mtnbikenc** [comment]
azure setup failed
/test azure-e2e-upgrade

**Comment by mtnbikenc** [comment]
There appears to be a problem in the azure-e2e-upgrade test.  It is failing consistently for this PR while waiting for workloads to scale.

File: docs/vsphere-golden-image.md [hunk 1]
```diff
@@ -93,13 +93,13 @@ In case no firewall rule exist, you must create it by running the following Powe
     New-NetFirewallRule -DisplayName 'OpenSSH Server (sshd)' -LocalPort 22 -Enabled True -Direction Inbound -Protocol TCP -Action Allow 
 ```
 
-## 4. Set up incoming connection for container logs
+## 4. Set up incoming connection for container logs and metrics
 
-Create a new firewall rule in the Windows VM to allow incoming connections for container logs, usually 
-on TCP port `10250` by running the following PowerShell command:
+Create new firewall rules in the Windows VM to allow incoming connections for container logs and metrics:
```
Comment (wgahnagl) [inline docs/vsphere-golden-image.md:98]: nit: would it be good to still call out the "usually on TCP port x" here?

**Review by wgahnagl — COMMENTED** [review]
/lgtm

**Pull request merged**